components/prism-ruby.js

```diff
@@ -5,17 +5,33 @@
  * 		constant, builtin, variable, symbol, regex
  */
 Prism.languages.ruby = Prism.languages.extend('clike', {
-	'comment': /#[^\r\n]*(\r?\n|$)/,
-	'keyword': /\b(alias|and|BEGIN|begin|break|case|class|def|define_method|defined|do|each|else|elsif|END|end|ensure|false|for|if|in|module|new|next|nil|not|or|raise|redo|require|rescue|retry|return|self|super|then|throw|true|undef|unless|until|when|while|yield)\b/,
+        'comment': /#(?!\{[^\r\n]*?\})[^\r\n]*(\r?\n|$)/g,
```

Comment:
Please use tabs to indent, not spaces.

Also, why the `g` flag everywhere? It is not used by Prism.